src/Inspetor/Exception/ModelException/InspetorEventException.php

```diff
@@ -30,11 +30,11 @@ class InspetorEventException extends ExceptionAbstract {
         3  => "timestamp is a required property. It can't be null.",
         4  => "creator_id is a required property. It can't be null on creation.",
         5  => "address is a required property when the event has physical place.  It can't be null on creation.",
-        6  => "sessions is a required property. It can't be null neither an empty array on creation.",
+        6  => "sessions is a required property. It must be an array of one or more sessions.",
         7  => "seating_options should be null or an array of strings.",
         8  => "categories should be null or an array of strings.",
         9  => "The status is not a valid one.",
```

Comment:
nit: I know it wasn't introduced now, but do you know if there's a reason why this is the only one out of these strings that start with a capital letter?

Reply:
I'm guessing it's because the other ones are properties of the `InspetorEvent` object. Seems to be a consistent convention at least (though unclear if preferable).

Reply:
Probably we just missed this one 😬